roles/elastic-stack/ansible-kibana/tasks/main.yml

```diff
@@ -41,7 +42,7 @@
   copy:
     src: "{{ item }}"
     dest: "{{ node_certs_destination }}/"
-    mode: '0664'
+    mode: 0444
```

Comment:
Read all permissions should not be assigned to a .key afaik

Reply:
Hello @jm404, this perm is now correctly set to `0440` on https://github.com/wazuh/wazuh-ansible/pull/382/commits/4b9fb53549acd8a0fd2712ce59953827a3125f05